pcweb/pages/docs/custom_components.py

```diff
@@ -186,7 +187,7 @@ def add_item(category: dict) -> rx.Component:
             updated_on_pypi_if_present(category["updated_on_pypi"]),
             source_if_present(category["source"]),
             demo_url_if_present(category["demo_url"]),
```

Comment:
Can we change this to just be an `rx.link(..., is_external=True)` instead of an iframe for now until we have time to do the iframe styling. We can just leave the iframe commented out for now